docker/install/ubuntu_install_zephyr.sh

```diff
@@ -69,10 +68,5 @@ chmod o+rwx zephyr/.cache
 #/opt/west/bin/pip3 install -r /opt/zephyrproject/zephyr/scripts/requirements.txt
 pip3 install -r /opt/zephyrproject/zephyr/scripts/requirements.txt
 
-ZEPHYR_SDK_VERSION=0.13.2
-ZEPHYR_SDK_FILE=zephyr-sdk-linux-setup.run
-wget --no-verbose -O $ZEPHYR_SDK_FILE \
-    https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v${ZEPHYR_SDK_VERSION}/zephyr-sdk-${ZEPHYR_SDK_VERSION}-linux-x86_64-setup.run
-chmod +x $ZEPHYR_SDK_FILE
-"./$ZEPHYR_SDK_FILE" -- -d /opt/zephyr-sdk
-rm "$ZEPHYR_SDK_FILE"
+ZEPHYR_INSTALL_SDK_SCRIPT=$(find -name "ubuntu_install_zephyr_sdk.sh")
+bash ${ZEPHYR_INSTALL_SDK_SCRIPT} /opt/zephyr-sdk
```

Comment:
I think you need to update ci_qemu to `ADD` this additional script, it'd also be better to explicitly `RUN` it rather than chaining them like this to make it clearer what gets run on build.

Reply:
Added to ci_qemu.
I agree that's preferred. But there are extra steps in `ubuntu_install_zephyr.sh` which are tied to the order of the execution. I suggest we keep it this way for now and do a refactor later. We need a bigger refactor to remove other duplications in building ci_qemu and Zephyr RVM.

Reply:
Sounds good!